telegram/_bot.py

```diff
@@ -2819,7 +2826,7 @@ def set_webhook(
         Args:
             url (:obj:`str`): HTTPS url to send updates to. Use an empty string to remove webhook
                 integration.
-            certificate (:obj:`filelike`): Upload your public key certificate so that the root
+            certificate (`filelike`): Upload your public key certificate so that the root
```

Comment:
try 
```suggestion
            certificate (:term:`file-object`): Upload your public key certificate so that the root
```

it's definitely defined in python [glossary ](https://docs.python.org/3/glossary.html#term-file-object) or if `:term:` doesn't work, try `:any:` [sphinx says](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-any) it should work...